Getting-Started/Code/Source-Control/index.md

```diff
@@ -0,0 +1,90 @@
+
+The strategy here will depend a little on which mode Modelsbuilder is running in.
+ - Pure Live, no source control allow Umbraco to generate
+ - App_Data, you can source control, and these will be built into your application dll
+ - Dll, if your working with a team, then this can be problematic managing conflicts, so remember to 'regenerate' your dll, after pulling from source control._
+ - API, generate the models into a seperate extension project and source control them.
+
+#### Media
+
+The Media section of Umbraco (unless configured otherwise) stores files in the /media folder, these can be updated by editors, in the Umbraco backoffice, so generally speaking you would not source control these files.
+
+#### Packages installed via Nuget
+
+The App_Plugins folder is the home for all third party packages installed on your site, if they are installed via NuGet then as long as the packages.config file in the route of your project is source controlled, then individual plugins shouldn't need to be source controlled - however each plugin could be different, and so may contain files it would be useful to track via Source control.
+
+### What folders and files should I Include in my source control repository?
+
+#### Front end build
+
+A lot depends on how you are running your front end build, eg you may be generating your css / js from SASS or CoffeeScript, and it is the source of your frontend implementation that would need to be source controlled, allowing the actual css/js files to be generated during the build process, but generally speaking it's your Javascript, Css and Page Furniture eg /img and fonts - anything your site implementation uses statically
+
+#### Views/Templates/Partials
+
+Umbraco site templates/views can be edited via the Umbraco Backoffice, but ultimately they reside in the views folder on disk, and should be source controlled.
+
+#### Controllers/Classes
+
+Any C# implementation, eg Surface Controllers, ViewModels etc, supporting projects, App_Code, Models generated by Modelsbuilder in API or App_Data mode - should be source controlled.
+
+#### Packages installed via the Backoffice
+
+The App_Plugins folder is the home for all third party packages installed on your site, if they are not installed via NuGet then they should be added to source control.
+
+#### Config
+
+Your site's /config folder contains the set of configuration files for your Umbraco site.
+
+#### DocumentType - Backoffice Structure Changes
+
+When you create and edit DocumentTypes, MediaTypes, DataTypes, Macros, DictionaryItems, Languages, MemberTypes & Templates('names' not the actual files) in the Umbraco Backoffice these values are stored in the Umbraco Database, making them difficult to source control. However there is a third party package called uSync - https://our.umbraco.org/projects/developer-tools/usync/ which when installed will serialize these settings to disk, in a folder called /uSync - enabling you to source control these changes too.
```

Comment:
We do also take this approach.  But I do have 2 remarks 

1) If we are mentioning a package like usync, shouldn't we say. a package like usync as there are other things out there? (which don't work/not work well/).  Courrier was serializing too right?  And you could say that modelsbuilder is also serializing (into c#).
2) is the default value "import on startup" still active?  Because that can cause serious side effects if you are deploying to another environment?